tests/setup

```diff
@@ -345,14 +345,20 @@ setup_db() {
         done
 
         echo "!$TESTCASE: waiting until ready"
+        echo "If we can't talk to any of the nodes the test will timeout"
         for node in $CLUSTER; do
-            echo "If we can't talk to any of the nodes the test will timeout"
-            out=$($CDB2SQL_EXE ${CDB2_OPTIONS} --tabs --host $node $DBNAME 'select 1' 2>&1)
-            while  [[ "$out" != "1" ]]; do
-                sleep 2
-                out=$($CDB2SQL_EXE ${CDB2_OPTIONS} --tabs --host $node $DBNAME 'select 1' 2>&1)
-                $CDB2SQL_EXE -v ${CDB2_OPTIONS} --tabs --host $node $DBNAME 'select 1' &> $LOGDIR/${DBNAME}.${node}.conn
+            tbl="dummy_$(cut -d'.' -f1 <<< $node)"
+            while :; do
+                $CDB2SQL_EXE ${CDB2_OPTIONS} --tabs --host $node $DBNAME "CREATE TABLE IF NOT EXISTS ${tbl}(i INTEGER)"
```

Comment:
Forgot to tell, could you also add a line of comment explaining why we are moving from 'select 1' to 'create table' for future reference?

Reply:
OK

Reply:
Change of plans - let me try first by tweaking initial connection back-off time. It was 5-10 seconds. Perhaps too much.